vendor/visualmetrics.py

```diff
@@ -1506,6 +1531,7 @@ def calculate_perceptual_speed_index(progress, directory):
     # Full Path of the Target Frame
     logging.debug("Target image for perSI is %s" % target_frame)
     ssim = ssim_1
+    value_frametime = []
```

Comment:
How about `completeness_percentages` here?

Reply:
I changed it to `completeness_value` because there are not percentages yet..